nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/NviService.java

```diff
@@ -67,7 +69,13 @@ private boolean exists(URI publicationId) {
 
     private void createCandidate(CandidateEvaluatedMessage evaluatedCandidate) {
         var pendingCandidate = toPendingCandidate(evaluatedCandidate.candidateDetails());
-        nviCandidateRepository.save(pendingCandidate);
+        var candidateDb = pendingCandidate.toDb();
+        nviCandidateRepository.save(candidateDb);
+    }
```

Comment:
This method should have Candidate as input parameter instead of CandidateEvaluateMessage